### ISSUE TITLE
Getting a Caused by: java.lang.ClassNotFoundException: deltaSharing.DefaultSource exception

### ISSUE THREAD

**Issue by tpang-cxl** [delta-io/delta-sharing#307]
got this exception when using the delta_sharing.load_as_spark() function, could you take a look at why this is happening? Thanks

**Comment by wchau** [comment]
Which version of the release are you using?

**Comment by wchau** [comment]
Have you tried installing delta sharing spark?

**Comment by linzhou-db** [comment]
@tpang-cxl

**Comment by tpang-cxl** [comment]
Yes I did. But I figured out that I need to add the deltasharing package
when In create the Spark instance:
.config("spark.jars.packages", "io.delta:delta-sharing-spark_2.12:0.6.4")
Thanks for getting back to me, you can close the ticket
